plugins/uniques.css

```diff
@@ -17,3 +17,7 @@
   margin-top: -5px;
   margin-bottom: -5px;
 }
+
+.ui-dialog-portalslist {
+  max-width: none !important;
+}
```

Comment:
?

Reply:
We have to widen the portal-list dialog to avoid a horizontal scroll bar.
it was fixed to 600px for no obvious reason.
This is the only place we found to be appropriate

Reply:
https://github.com/IITC-CE/ingress-intel-total-conversion/blob/67fcdf9adb8c079de6332742f12df7c99f4b91bb/core/style.css#L1036-L1044

Don't use `!important`
prefer use more specific rules like 
```css
.ui-dialog.ui-dialog-portallist {
  max-width: max-width: calc(100vw - 2px);
}
```

or open a discussion about the 600px

Reply:
trying that